Use extract/inject in saved query service

lukasolson opened the pull request:
## Summary

Resolves https://github.com/elastic/kibana/issues/108184.

Uses `filterManager.inject`/`extract` inside the saved query service to properly inject/extract references to index patterns (if filters inside the saved query reference them).

### Checklist

- [x] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios

### For maintainers

- [x] This was checked for breaking API changes and was [labeled appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

### Release note

Saved query saved objects now properly reference index pattern saved objects, so they can be exported/imported properly.

elasticmachine commented:
Pinging @elastic/kibana-app-services (Team:AppServices)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/)
* Commit: 43a0011cf299416e833b1c0e95b45c23b9f569a2
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types and Public API Docs](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/execution/node/428/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/","number":"151082","commit":"43a0011cf299416e833b1c0e95b45c23b9f569a2"}]}
PIPELINE-->

ppisljar commented on src/plugins/data/public/query/saved_query/saved_query_service.ts:
```diff
@@ -19,7 +24,8 @@ type SerializedSavedQueryAttributes = SavedObjectAttributes &
 export const createSavedQueryService = (
-  savedObjectsClient: SavedObjectsClientContract
+  savedObjectsClient: SavedObjectsClientContract,
+  filterManager: FilterManager
```
instead of providing filterManager contract here, you could import filter manager extract/inject utilities (as they are stateless)

`import { extract, inject } from '../../common/query/persistable_state';`

ppisljar commented on src/plugins/data/public/query/saved_query/saved_query_service.ts:
```diff
@@ -41,8 +47,11 @@ export const createSavedQueryService = (
       query,
     };
 
+    let references = [];
```
i would extract this into `common/query/saved_query/persistable_state` as it needs to be exposed on the server as well.

ppisljar replied:
or common/query/persistableState (where filters utilities already are)

ppisljar replied:
and please expose on the server as well.

lukasolson replied:
Would it make sense to create a dedicated service with routes on the server? And then the saved query service on the client would just make HTTP calls and this logic would all be done on the server instead? I'm starting to go down this path, let me know if you disagree with this approach.

ppisljar reviewed (changes requested):
can we also update saved queries saved object at the same time to use migrations correctly ?
```
migrations: () => {
  return mapValues(filterManager.getAllMigrations(), (migrate) => {
        return (state: SavedObjectUnsanitizedDoc) => ({
          ...state,
          attributes: {
            ...state.attributes,
            filters: migrate(state.attributes.filters),
          }
        });
      });
    },
}
```

lukasolson commented:
@ppisljar Could you take a look at the [latest commit](https://github.com/elastic/kibana/pull/111213/commits/eb0a9ddc54acb94455e0886def4c1efa77fcbcc0)? I've moved most of the logic to a route handler context (so that any consumers can use `context.savedQuery.create`/etc. directly). I've also exposed routes that call these functions and adjusted the client-side service to call these routes (instead of interacting directly with the saved objects client). Let me know your thoughts.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/)
* Commit: eb0a9ddc54acb94455e0886def4c1efa77fcbcc0
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Doc API Changes](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/execution/node/425/log)

### History
* :broken_heart: [Build #151082](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/) failed 43a0011cf299416e833b1c0e95b45c23b9f569a2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/","number":"151082","commit":"43a0011cf299416e833b1c0e95b45c23b9f569a2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/","number":"152420","commit":"eb0a9ddc54acb94455e0886def4c1efa77fcbcc0"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/)
* Commit: 4950982fd1e4f302c2837bd60c01314c0e33146a
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Doc API Changes](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/execution/node/425/log)

### History
* :broken_heart: [Build #152662](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/) failed 7c545a11b4e210334108e441d0d28eb282d86799
* :broken_heart: [Build #152420](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/) failed eb0a9ddc54acb94455e0886def4c1efa77fcbcc0
* :broken_heart: [Build #151082](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/) failed 43a0011cf299416e833b1c0e95b45c23b9f569a2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/","number":"151082","commit":"43a0011cf299416e833b1c0e95b45c23b9f569a2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/","number":"152420","commit":"eb0a9ddc54acb94455e0886def4c1efa77fcbcc0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/","number":"152662","commit":"7c545a11b4e210334108e441d0d28eb282d86799"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/","number":"153026","commit":"4950982fd1e4f302c2837bd60c01314c0e33146a"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/)
* Commit: 075c7122db588dbc7eaedc4bd4364cbdb7552517
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Doc API Changes](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/execution/node/425/log)

### History
* :broken_heart: [Build #153026](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/) failed 4950982fd1e4f302c2837bd60c01314c0e33146a
* :broken_heart: [Build #152662](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/) failed 7c545a11b4e210334108e441d0d28eb282d86799
* :broken_heart: [Build #152420](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/) failed eb0a9ddc54acb94455e0886def4c1efa77fcbcc0
* :broken_heart: [Build #151082](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/) failed 43a0011cf299416e833b1c0e95b45c23b9f569a2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/","number":"151082","commit":"43a0011cf299416e833b1c0e95b45c23b9f569a2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/","number":"152420","commit":"eb0a9ddc54acb94455e0886def4c1efa77fcbcc0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/","number":"152662","commit":"7c545a11b4e210334108e441d0d28eb282d86799"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/","number":"153026","commit":"4950982fd1e4f302c2837bd60c01314c0e33146a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/","number":"154418","commit":"075c7122db588dbc7eaedc4bd4364cbdb7552517"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155913/)
* Commit: 3687dbe2855f006a4062836425f3b2a564fa0ad7
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Doc API Changes](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155913/execution/node/425/log)

### History
* :broken_heart: [Build #154418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/) failed 075c7122db588dbc7eaedc4bd4364cbdb7552517
* :broken_heart: [Build #153026](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/) failed 4950982fd1e4f302c2837bd60c01314c0e33146a
* :broken_heart: [Build #152662](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/) failed 7c545a11b4e210334108e441d0d28eb282d86799
* :broken_heart: [Build #152420](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/) failed eb0a9ddc54acb94455e0886def4c1efa77fcbcc0
* :broken_heart: [Build #151082](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/) failed 43a0011cf299416e833b1c0e95b45c23b9f569a2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151082/","number":"151082","commit":"43a0011cf299416e833b1c0e95b45c23b9f569a2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/","number":"152420","commit":"eb0a9ddc54acb94455e0886def4c1efa77fcbcc0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/","number":"152662","commit":"7c545a11b4e210334108e441d0d28eb282d86799"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/","number":"153026","commit":"4950982fd1e4f302c2837bd60c01314c0e33146a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/","number":"154418","commit":"075c7122db588dbc7eaedc4bd4364cbdb7552517"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155913/","number":"155913","commit":"3687dbe2855f006a4062836425f3b2a564fa0ad7"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160719/)
* Commit: 7c42cd453c7dc990f0539a60a2d98e7324a5b404
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Doc API Changes](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160719/execution/node/425/log)

### History
* :broken_heart: [Build #155913](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155913/) failed 3687dbe2855f006a4062836425f3b2a564fa0ad7
* :broken_heart: [Build #154418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/) failed 075c7122db588dbc7eaedc4bd4364cbdb7552517
* :broken_heart: [Build #153026](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/) failed 4950982fd1e4f302c2837bd60c01314c0e33146a
* :broken_heart: [Build #152662](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/) failed 7c545a11b4e210334108e441d0d28eb282d86799
* :broken_heart: [Build #152420](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/) failed eb0a9ddc54acb94455e0886def4c1efa77fcbcc0

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152420/","number":"152420","commit":"eb0a9ddc54acb94455e0886def4c1efa77fcbcc0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/152662/","number":"152662","commit":"7c545a11b4e210334108e441d0d28eb282d86799"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/153026/","number":"153026","commit":"4950982fd1e4f302c2837bd60c01314c0e33146a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/","number":"154418","commit":"075c7122db588dbc7eaedc4bd4364cbdb7552517"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155913/","number":"155913","commit":"3687dbe2855f006a4062836425f3b2a564fa0ad7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160719/","number":"160719","commit":"7c42cd453c7dc990f0539a60a2d98e7324a5b404"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162337/)
* Commit: b992e477d5f596c3684d6a93d23223b61da17078
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Build Kibana](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162337/execution/node/371/log)

### History
* :broken_heart: [Build #162250](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162250/) failed 89356609c33d3df866b15dbeae736fe360eec9d6
* :broken_heart: [Build #161196](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161196/) failed fa3053c4a0b5b181b47f3e6135e1a3fe0f775231
* :broken_heart: [Build #160719](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160719/) failed 7c42cd453c7dc990f0539a60a2d98e7324a5b404
* :broken_heart: [Build #155913](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155913/) failed 3687dbe2855f006a4062836425f3b2a564fa0ad7
* :broken_heart: [Build #154418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/) failed 075c7122db588dbc7eaedc4bd4364cbdb7552517

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/154418/","number":"154418","commit":"075c7122db588dbc7eaedc4bd4364cbdb7552517"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/155913/","number":"155913","commit":"3687dbe2855f006a4062836425f3b2a564fa0ad7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160719/","number":"160719","commit":"7c42cd453c7dc990f0539a60a2d98e7324a5b404"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161196/","number":"161196","commit":"fa3053c4a0b5b181b47f3e6135e1a3fe0f775231"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162250/","number":"162250","commit":"89356609c33d3df866b15dbeae736fe360eec9d6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162337/","number":"162337","commit":"b992e477d5f596c3684d6a93d23223b61da17078"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/163034/)
* Commit: 48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-111213/48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/2aaa515bbc6d40688b1a40f24295f79946fc09e3) | [after](https://github.com/elastic/kibana/commit/48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f) | diff |
| --- | --- | --- | --- |
| `data` | 534 | 533 | -1 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/2aaa515bbc6d40688b1a40f24295f79946fc09e3) | [after](https://github.com/elastic/kibana/commit/48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f) | diff |
| --- | --- | --- | --- |
| `data` | 2791 | 2802 | +11 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/2aaa515bbc6d40688b1a40f24295f79946fc09e3) | [after](https://github.com/elastic/kibana/commit/48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f) | diff |
| --- | --- | --- | --- |
| `data` | 102.7KB | 102.9KB | +245.0B |
| `securitySolution` | 4.6MB | 4.6MB | -34.0B |
| total |  |  | +211.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/2aaa515bbc6d40688b1a40f24295f79946fc09e3) | [after](https://github.com/elastic/kibana/commit/48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f) | diff |
| --- | --- | --- | --- |
| `data` | 465.8KB | 464.7KB | -1.1KB |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/2aaa515bbc6d40688b1a40f24295f79946fc09e3) | [after](https://github.com/elastic/kibana/commit/48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f) | diff |
| --- | --- | --- | --- |
| `data` | 3181 | 3192 | +11 |


</details>

### History
* :broken_heart: [Build #162876](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162876/) failed 6f4199ac151a57f72ccedb2a9a0f0232ec576da3
* :broken_heart: [Build #162337](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162337/) failed b992e477d5f596c3684d6a93d23223b61da17078
* :broken_heart: [Build #162250](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162250/) failed 89356609c33d3df866b15dbeae736fe360eec9d6
* :broken_heart: [Build #161196](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161196/) failed fa3053c4a0b5b181b47f3e6135e1a3fe0f775231
* :broken_heart: [Build #160719](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160719/) failed 7c42cd453c7dc990f0539a60a2d98e7324a5b404

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lukasolson

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160719/","number":"160719","commit":"7c42cd453c7dc990f0539a60a2d98e7324a5b404"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161196/","number":"161196","commit":"fa3053c4a0b5b181b47f3e6135e1a3fe0f775231"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162250/","number":"162250","commit":"89356609c33d3df866b15dbeae736fe360eec9d6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162337/","number":"162337","commit":"b992e477d5f596c3684d6a93d23223b61da17078"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/162876/","number":"162876","commit":"6f4199ac151a57f72ccedb2a9a0f0232ec576da3"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/163034/","number":"163034","commit":"48c2c6d92349c68cdc0a9c87f8946b6b19a7f21f"}]}
PIPELINE-->

kibanamachine commented:
## 💚 Backport successful

| Status | Branch | Result |
|:------:|:------:|:------:|
| ✅ |  [7.16](https://github.com/elastic/kibana/pull/115978)  | [<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/115978">](https://github.com/elastic/kibana/pull/115978) |

This backport PR will be merged automatically after passing CI.
